frontend/src/components/ToastMessageNotification.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import { notificationIcons } from '@/app/svgIcons'
+interface AlertNameProp{
+    alertName: "alert alert-info"| "alert alert-success" | "alert alert-error" | "alert alert-warning"
```

Comment:
I think you are prepending alert to the alert name down below. We could also added the alert- to it there as well. This means the alert name could just be info, success, error and warning which is much cleaner.

Reply:
I was so far gone I didn't even realize what I left in there. Reason 2/2 for not programing that late at night. I've applied what you suggested with a slightly different approach. Doing const myThing = "alert alert-"+stringName 
doesn't work for some weird reason. I just don't get any styling. I tried it in production, and I was at a loss, so the current changes do solve that problem.